.gitpod.dockerfile

```diff
@@ -3,6 +3,9 @@ FROM gitpod/workspace-postgres
 # Install Ruby
 ENV RUBY_VERSION=3.0.2
 
+# Install the GitHub CLI
+RUN brew install gh
```

Comment:
From https://www.gitpod.io/blog/brew#you-can-now-brew-install-anything-in-gitpod-

> brew installs everything under /home/linuxbrew/, but Gitpod only backs up your Docker image plus everything inside /workspace/. This means that if you brew install fzf in a Gitpod terminal, and then your workspace stops running, when you restart it later the package fzf will be gone. If that’s a problem, simply move the brew install fzf command into your .gitpod.Dockerfile to make it permanent.